docs/user/tutorial.rst

```diff
@@ -55,7 +55,7 @@ Now, open ``app.py`` in your favorite text editor and add the following lines:
 
     import falcon
 
-    api = application = falcon.API()
+    api = application = falcon.App()
```

Comment:
Maybe we should just standardize on `app` here as well?

Reply:
Agree